infrastructure/kube/templates/electrs/initcontainer/provision-electrs/provision-electrs.js

```diff
@@ -0,0 +1,66 @@
+const fs = require('fs')
+const toml = require("toml")
+const tomlify = require("tomlify-j0.4")
+
+const bcoinUrl = process.env.BITCOIN_URL
+const jsonRpcImport = process.env.JSON_RPC_IMPORT
+const dbDir = process.env.DB_DIR
+const network = process.env.NETWORK
+const electrumRpcAddress = process.env.ELECTRUM_RPC_ADDRESS
+const monitoringAddress = process.env.MONITORING_ADDRESS
+const verbose = process.env.VERBOSE
+
+async function provisionElectrs() {
+    console.log('###########  Provisioning electrs! ###########')
+
+    console.log('\n<<<<<<<<<<<< Creating electrs config file >>>>>>>>>>>>')
+    await createElectrsConfig()
+
+    console.log("\n########### Electrs provisioning complete! ###########")
+}
+
+async function createElectrsConfig() {
+    const configFile = toml.parse(
+        fs.readFileSync("/tmp/electrs-config.toml", "utf8")
+    )
+
+    // Bitcoin url has the following format: `http://username:password@address`
+    // Find position of `@` to extract `auth`(`username:password`) and `address`
+    const atIdx = bcoinUrl.indexOf("@", 7)
```

Comment:
Worth extracting `7` to a self-explaining constant. Also, what if we use a `https` address?

Reply:
I changed it so it works with both `http` and `https`: https://github.com/keep-network/electrs/pull/1/commits/b04ecc77b7be1e4024bcc5ca64d662fa584b79ed